README.md

```
@@ -23,15 +23,13 @@ One of the standout features of this application is its offline capability, allo
 
 > \[!Note]
 > **We're moving towards to Jira for issue tracking. Please use [Jira](https://mifosforge.jira.com/jira/software/c/projects/MIFOSAC/issues/) for issue tracking.**
-> **And Join our [slack](https://join.slack.com/t/mifos/shared_invite/zt-2wvi9t82t-DuSBdqdQVOY9fsqsLjkKPA) community channel `android-client` to discuss all things about Android Client development. Please keep discussions focused and avoid cross-posting across channels.**
+> **And Join our [slack](https://join.slack.com/t/mifos/shared_invite/zt-2wvi9t82t-DuSBdqdQVOY9fsqsLjkKPA) community channel `mifos-x-field-officer-app` to discuss all things about Android Client development. Please keep discussions focused and avoid cross-posting across channels.**
 
-## Demo credentials
-- **Fineract Instance**: `gsoc.mifos.community`
-- **Username**: `mifos`
-- **Password**: `password`
+### Demo Credentials
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)